tests/lib/bootstrap.js

```diff
@@ -68,6 +69,10 @@ function requireTests( dependencies, noBackCompat ) {
 		dependencies.push( "testswarm" );
 	}
 
+	if ( parseUrl().migrate ) {
```

Comment:
This isn't actually working because `parseUrl()` doesn't support booleans right now.